data/initrd/scripts/url/hmc/mount

```diff
@@ -0,0 +1,40 @@
+#! /bin/bash
+
+# https://github.com/openSUSE/linuxrc/blob/master/user_defined_url_schemes.txt
+
+set -e
+
+echo "url_dir = '$url_dir'"
+
+modprobe hmcdrv
+
+if [ -d "$url_dir" ] ; then
+  hmcdrvfs "$url_dir"
+
```

Comment:
Note the 'set -e'

It's not the task of this script to verify anything. linuxrc will do that. This script is only there to provide the mount.